docs/actions-new.md

```diff
@@ -146,7 +146,8 @@ The initialization route is `/init`. It must accept a `POST` request with a JSON
     "name" : String,
     "main" : String,
     "code" : String,
-    "binary": Boolean
+    "binary": Boolean,
+    "env": Map[String, JSON Value]
```

Comment:
Ideally it should be a map of `<String, String>`. This would simplify the implementation to just handle the value directly. Otherwise they would need to stringify the value which may vary depending on runtime

Reply:
OK - I thought I'd defer this to the runtime so the invoker is doing less work but I can make this change, as your point is valid.

Reply:
Thinking about this further, if the value is JSON stringified in a runtime-specific way, it should still be JSON parsed back to the original value. To do this on the invoker side means walking the properties and normalizing each property individually. It would indeed be less work for the runtimes but more work on the invoker. Thoughts?

Reply:
The last commit implement normalization of the env vars to a map[string, string].

Reply:
>  It would indeed be less work for the runtimes but more work on the invoker

Yes but then we would only need to get it right at one place and have consistent behavior for various runtimes

Thanks for addressing this.